_layouts/page.html

```diff
@@ -3,6 +3,15 @@
 ---
 <!-- {% include breadcrumbs.html %} -->
 <article class="article">
+  <!-- {% if page.development %} -->
+  <div class="grid-block well gray well-in-development">
+    <img class="grid-block shrink icon" src="{{ site.baseurl }}/images/in-development.svg" width="45" height="45">
+    <div class="grid-block vertical">
+      <div class="weight-strong">In Development</div>
+      <div class="weight-light">We’re hard at work on this feature. <br class="hidden-sm">You can expect it soon!</div>
```

Comment:
I think I prefer this without the last line.

<img width="864" alt="screen shot 2017-05-17 at 11 06 17 am" src="https://cloud.githubusercontent.com/assets/5341618/26168962/e3ae11fc-3af0-11e7-9d14-33f203c721b7.png">